clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -6867,6 +6893,17 @@ ExprResult Sema::ActOnOpenMPCall(ExprResult Call, Scope *Scope,
   if (!CalleeFnDecl)
     return Call;
 
+  if (this->LangOpts.OpenMP >= 51 && CalleeFnDecl->getIdentifier()) {
+    if (CalleeFnDecl->getName().startswith_insensitive(StringRef("omp_"))) {
```

Comment:
The two `if` conditions can be combined:

```suggestion
  if (this->LangOpts.OpenMP >= 51 && CalleeFnDecl->getIdentifier() &&
      CalleeFnDecl->getName().startswith_insensitive(StringRef("omp_")) {
```